packages/bot-web-ui/src/components/dashboard/dashboard.scss

```diff
@@ -1,7 +1,76 @@
+@import '../dashboard/dashboard-components/index.scss';
+
 .dashboard {
     &__container {
-        padding: 1.2rem;
-        max-width: 100rem;
-        border: solid 1px var(--general-section-1);
+        position: relative;
+    }
+    &__chart-wrapper {
+        position: relative;
+        height: calc(100vh - 150px);
+    }
+    &__toolbox {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 23.6rem;
+        padding: 0.8rem;
+        background: var(--general-main-1);
+        .db-toolbox {
+            &__title {
+                height: 2.6rem;
+                line-height: 2.6rem;
+                position: relative;
+                cursor: pointer;
+                &-chevron {
```

Comment:
maybe &--?

Reply:
fixed it... made is a section __chevron as its a sub section of title ✅